src/mscorlib/src/System/IO/Stream.cs

```diff
@@ -172,57 +160,19 @@ public virtual Task CopyToAsync(Stream destination, Int32 bufferSize, Cancellati
         // Reads the bytes from the current stream and writes the bytes to
         // the destination stream until all bytes are read, starting at
         // the current position.
-        public void CopyTo(Stream destination)
-        {
-            if (destination == null)
-                throw new ArgumentNullException("destination");
-            if (!CanRead && !CanWrite)
-                throw new ObjectDisposedException(null, Environment.GetResourceString("ObjectDisposed_StreamClosed"));
-            if (!destination.CanRead && !destination.CanWrite)
-                throw new ObjectDisposedException("destination", Environment.GetResourceString("ObjectDisposed_StreamClosed"));
-            if (!CanRead)
-                throw new NotSupportedException(Environment.GetResourceString("NotSupported_UnreadableStream"));
-            if (!destination.CanWrite)
-                throw new NotSupportedException(Environment.GetResourceString("NotSupported_UnwritableStream"));
-            Contract.EndContractBlock();
-
-            InternalCopyTo(destination, _DefaultCopyBufferSize);
-        }
+        public void CopyTo(Stream destination) =>
```

Comment:
It would look better to do it old fashioned way to maintain consistent style within the file.